File: workflow/src/python/10_analysis.py
from pathlib import Path

ROOT = Path(__file__).resolve().parents[2]
RAW = ROOT / "data" / "raw"
PROC = ROOT / "data" / "processed"
OUT_TABLES = ROOT / "output" / "tables"
OUT_FIGS = ROOT / "output" / "figures"

for p in (PROC, OUT_TABLES, OUT_FIGS):
    p.mkdir(parents=True, exist_ok=True)

# TODO: load data and run analysis

File: workflow/src/matlab/10_analysis.m
% 10_analysis.m
root = fileparts(fileparts(fileparts(mfilename('fullpath'))));
dataRaw = fullfile(root, 'data', 'raw');
dataProcessed = fullfile(root, 'data', 'processed');
outTables = fullfile(root, 'output', 'tables');
outFigures = fullfile(root, 'output', 'figures');

if ~exist(dataProcessed, 'dir'); mkdir(dataProcessed); end
if ~exist(outTables, 'dir'); mkdir(outTables); end
if ~exist(outFigures, 'dir'); mkdir(outFigures); end

% TODO: load data and run analysis
